src/main/java/org/sagebionetworks/web/client/jsinterop/TwoFactorAuthSettingsPanelProps.java

```diff
@@ -15,15 +15,18 @@ public interface Callback {
 
   Callback onBeginTwoFactorEnrollment;
   Callback onRegenerateBackupCodes;
+  boolean hideTitle;
 
   @JsOverlay
   public static TwoFactorAuthSettingsPanelProps create(
     Callback onBeginTwoFactorEnrollment,
     Callback onRegenerateBackupCodes
   ) {
-    TwoFactorAuthSettingsPanelProps props = new TwoFactorAuthSettingsPanelProps();
+    TwoFactorAuthSettingsPanelProps props =
+      new TwoFactorAuthSettingsPanelProps();
     props.onBeginTwoFactorEnrollment = onBeginTwoFactorEnrollment;
     props.onRegenerateBackupCodes = onRegenerateBackupCodes;
+    props.hideTitle = true; // always true in SWC
```

Comment:
New prop, so we will have to bump SRC: https://github.com/Sage-Bionetworks/synapse-web-monorepo/pull/677